rules/framework/vfs_overlay.bzl

```diff
@@ -360,8 +359,12 @@ def make_vfsoverlay(ctx, hdrs, module_map, private_hdrs, has_swift, swiftmodules
         "version": 0,
         "case-sensitive": True,
         "overlay-relative": True,
-        "use-external-names": False,
-        "roots": roots,
+        "use-external-names": True,
```

Comment:
can explain a bit / point to doc explaining what does this boolean do? Tried to find its doc but can only fine VirtualFileSystem.h doc for it :pray: @jerrymarino